compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmDesugarPhase.java

```diff
@@ -160,275 +154,120 @@ private static void rewriteRecordInitFunction(BIRFunction func, BRecordType reco
-            globalVar.name = Names.fromString(encodeNonFunctionIdentifier(globalVar.name.value, encodedVsInitialIds));
+            globalVar.name = Names.fromString(encodeNonFunctionIdentifier(globalVar.name.value));
         }
     }
 
-    // Replace encoding identifiers
-    static void replaceEncodedModuleIdentifiers(BIRNode.BIRPackage module,
-                                                HashMap<String, String> encodedVsInitialIds) {
-        replaceEncodedPackageIdentifiers(module.packageID, encodedVsInitialIds);
-        replaceEncodedGlobalVariableIdentifiers(module.globalVars, encodedVsInitialIds);
-        replaceEncodedImportedGlobalVariableIdentifiers(module.importedGlobalVarsDummyVarDcls, encodedVsInitialIds);
-        replaceEncodedFunctionIdentifiers(module.functions, encodedVsInitialIds);
-        replaceEncodedTypeDefIdentifiers(module.typeDefs, encodedVsInitialIds);
-    }
-
-    private static void replaceEncodedPackageIdentifiers(PackageID packageID,
-                                                         HashMap<String, String> encodedVsInitialIds) {
-        packageID.orgName = getInitialIdString(packageID.orgName, encodedVsInitialIds);
-        packageID.name = getInitialIdString(packageID.name, encodedVsInitialIds);
-    }
-
-    private static void replaceEncodedTypeDefIdentifiers(List<BIRTypeDefinition> typeDefs,
-                                                         HashMap<String, String> encodedVsInitialIds) {
-        for (BIRTypeDefinition typeDefinition : typeDefs) {
-            typeDefinition.type.tsymbol.name = getInitialIdString(typeDefinition.type.tsymbol.name,
-                    encodedVsInitialIds);
-            typeDefinition.internalName = getInitialIdString(typeDefinition.internalName, encodedVsInitialIds);
-            replaceEncodedFunctionIdentifiers(typeDefinition.attachedFuncs, encodedVsInitialIds);
-            BType bType = JvmCodeGenUtil.getImpliedType(typeDefinition.type);
-            if (bType.tag == TypeTags.OBJECT) {
-                BObjectType objectType = (BObjectType) bType;
-                BObjectTypeSymbol objectTypeSymbol = (BObjectTypeSymbol) bType.tsymbol;
-                if (objectTypeSymbol.attachedFuncs != null) {
-                    replaceEncodedAttachedFunctionIdentifiers(objectTypeSymbol.attachedFuncs, encodedVsInitialIds);
-                }
-                for (BField field : objectType.fields.values()) {
-                    field.name = getInitialIdString(field.name, encodedVsInitialIds);
-                }
-            }
-            if (bType.tag == TypeTags.RECORD) {
-                BRecordType recordType = (BRecordType) bType;
-                for (BField field : recordType.fields.values()) {
-                    field.name = getInitialIdString(field.name, encodedVsInitialIds);
-                }
-            }
-        }
+    private static String encodeFunctionIdentifier(String identifier) {
```

Comment:
As this function only contains a static method call, we can remove this and use the `Utils.encodeFunctionIdentifier(identifier)` directly.